src/wp-includes/class-wpdb.php

```diff
@@ -3950,14 +3779,12 @@ public function get_charset_collate() {
 	 * @since 2.7.0
 	 * @since 4.1.0 Added support for the 'utf8mb4' feature.
 	 * @since 4.6.0 Added support for the 'utf8mb4_520' feature.
-	 * @since 6.1.0 Added support for the 'identifier_placeholders' feature.
 	 *
 	 * @see wpdb::db_version()
 	 *
 	 * @param string $db_cap The feature to check for. Accepts 'collation', 'group_concat',
-	 *                       'subqueries', 'set_charset', 'utf8mb4', 'utf8mb4_520',
-	 *                       or 'identifier_placeholders'.
-	 * @return bool True when the database feature is supported, false otherwise.
```

Comment:
The opening description for this DocBlock was also changed in the changeset https://core.trac.wordpress.org/changeset/53575#file0

Before the r53575, it read:

```php
/**
  * Determines if a database supports a particular feature.
  *
```

@SergeyBiryukov what do you think?

Reply:
I left it as is, since it was clarified and does not seem to be particularly affected by the revert.